app/Routes/RouteManager.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Routes;
+
+use App\Controllers\WordController;
+use App\Entities\Response;
+use App\Exception\NotFoundException;
+use App\Repositories\WordRepository;
+
+readonly class RouteManager
+{
+    /** @var Route[] $routes */
+    private array $routes;
+
+    public function __construct(
+        private WordRepository $wordRepository,
+    ) {
+        $wordController = new WordController($wordRepository);
+
+        $this->routes = [
+            new Route('GET', '/words', $wordController, 'getAll'),
+            new Route('GET', '/words/{id}', $wordController, 'getById'),
+            new Route('POST', '/words', $wordController, 'create'),
+            new Route('PUT', '/words/{id}', $wordController, 'update'),
+            new Route('DELETE', '/words/{id}', $wordController, 'delete'),
+        ];
```

Comment:
__optional__ but recomended

improvement - pass these values as dependencies this way service is less coupled and more configurable (you could have 2 different instances with 2 different array of routes)

Reply:
Also I would put methods into ENUM or CONST some sort of. However __optional__